invenio_rdm_migrator/streams/communities/models.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Dataclasses user models to generate table rows."""
+
+from dataclasses import InitVar, dataclass
+
+
+@dataclass
+class Community:
+    """Community dataclass model."""
+
+    id: str
+    created: str
+    updated: str
+    json: dict
+    version_id: int
+    slug: str
+    bucket_id: str
+
+    _table_name: InitVar[str] = "communities_metadata"
+
+
+@dataclass
+class CommunityMembers:
```

Comment:
```suggestion
class CommunityMember:
```

Singular for consistency